model_compression_toolkit/qat/keras/quantizer/lsq/symmetric_lsq.py

```diff
@@ -87,12 +87,23 @@ def __init__(self, quantization_config: TrainableQuantizerWeightsConfig):
         self.threshold_shape = self.threshold_values.shape
         self.per_channel = self.quantization_config.weights_per_channel_threshold
         self.channel_axis = self.quantization_config.weights_channels_axis
+
+        if self.per_channel and self.channel_axis not in [-1, len(self.threshold_shape) - 1]:
```

Comment:
you're not using `fake_quant_with_min_max_vars_per_channel` so this is an overkill. Why not just replace line 145 with:
`thresholds = tf.reshape(self.get_quantizer_variable(THRESHOLD_TENSOR), self.threshold_shape)`